client/src/main/java/org/spine3/time/ZoneOffsets.java

```diff
@@ -52,6 +54,16 @@ private ZoneOffsets() {
     }
 
     /**
+     * Obtains the ZoneOffset instance using timezone offset of the Java virtual machine.
+     */
+    public static ZoneOffset getDefault() {
+        final int seconds = TimeZone.getDefault().getRawOffset()/1000;
```

Comment:
I still have the same question. What I can see now in this line is 
```java
final int seconds = TimeZone.getDefault().getRawOffset()/1000;
```
And `1000` is still just a number with no explanation.